### PR TITLE
Preserves position of SplitPane from redux store

### PR DESCRIPTION
**Pull request by anmolarora1**

### CODE REVIEW

File: src/action-creators/splitView.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export const updateSplitPosition = splitPosition => dispatch => {
```
Comment (raineorshine) [inline src/action-creators/splitView.js]: Please match the filename with the action-creator name.

File: src/reducers/splitView.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+export const updateSplitPosition = (state, { value }) => {
+  localStorage.setItem('splitPosition', parseInt(value, 10))
```
Comment (raineorshine) [inline src/reducers/splitView.js]: There shouldn't be any side effects in reducers. I believe this should go in the action-creator thunk.

File: src/reducers/splitView.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+  return { ...state, splitPosition: value }
+}
+
+export const toggleSplitView = (state, { value }) => {
```
Comment (raineorshine) [inline src/reducers/splitView.js]: There should be one reducer per file, and the filename should match the reducer name. This makes it easier find reducer definitions with some editors.

File: src/action-creators/splitView.js (hunk 1 above)
Comment (anmolarora1) [reply src/action-creators/splitView.js]: Got it. I'll update it

File: src/reducers/splitView.js (hunk 2 above)
Comment (anmolarora1) [reply src/reducers/splitView.js]: Thanks. I'll change it

File: src/reducers/splitView.js (hunk 3 above)
Comment (anmolarora1) [reply src/reducers/splitView.js]: We might end up with too many files in that case. Maybe we can couple the reducers together into the files based on their functionalities/components using them. Just a thought.  
I'll update this for now though.

**Comment by raineorshine** [comment]
Also, it appears there is still a merge conflict.

File: src/reducers/splitView.js (hunk 3 above)
Comment (raineorshine) [reply src/reducers/splitView.js]: Great, we'll keep an eye on it. I haven't experienced this to be a problem yet. I find the naming convention saves a lot of time.

**Comment by anmolarora1** [comment]
Creating a new pr because the commits are overlapping due to incorrect rebasing. I'll be more careful about this in future

**Pull request closed without merging**